src/lxc/conf.c

```diff
@@ -1235,7 +1235,7 @@ static int lxc_fill_autodev(const struct lxc_rootfs *rootfs)
 	if (!dir_exists(path))
 		return 0;
 
-	INFO("Populating \"/dev\"");
+	INFO("Populating devtmpfs");
```

Comment:
It's specifically NOT a devtmpfs though, right?

Reply:
Technically it's a tmpfs, yes. I just thought that devtmpfs is nicer. But I can update or you can merge now and I'll send a small update later